ios/NotificationService/NotificationService.swift

```diff
@@ -75,6 +79,44 @@ class NotificationService: UNNotificationServiceExtension {
     }
   }
   
+  private func sendInvalidNotificationIntent() {
+    guard let notification = bestAttemptContent else { return }
+    if #available(iOSApplicationExtension 15.0, *) {
+      os_log(OSLogType.default, "Mattermost Notifications: creating invalid intent")
+      
+      bestAttemptContent?.body = "We could not verify this notification with the server"
+      bestAttemptContent?.userInfo.updateValue("false", forKey: "verified")
+      let intent = INSendMessageIntent(recipients: nil,
+                                       outgoingMessageType: .outgoingMessageText,
+                                       content: "We could not verify this notification with the server",
+                                       speakableGroupName: nil,
+                                       conversationIdentifier: "NOT_VERIFIED",
+                                       serviceName: nil,
+                                       sender: nil,
+                                       attachments: nil)
+      
+      let interaction = INInteraction(intent: intent, response: nil)
+      interaction.direction = .incoming
+      interaction.donate { error in
+        if error != nil {
+          self.contentHandler?(notification)
+          os_log(OSLogType.default, "Mattermost Notifications: sendMessageIntent intent error %{public}@", error! as CVarArg)
+        }
+        
+        do {
+          let updatedContent = try notification.updating(from: intent)
+          os_log(OSLogType.default, "Mattermost Notifications: present updated notification")
+          self.contentHandler?(updatedContent)
+        } catch {
+          os_log(OSLogType.default, "Mattermost Notifications: something failed updating the notification %{public}@", error as CVarArg)
+          self.contentHandler?(notification)
+        }
+      }
+    } else {
+      self.contentHandler?(notification)
```

Comment:
This will display the original msg in the notification for OS lower than 15, we ok with that? That if of course unless you address the previous comment